pkg/standalone-utils/contracts/BatchRelayerLibrary.sol

```diff
@@ -21,13 +21,14 @@ import "./relayer/AaveWrapping.sol";
 import "./relayer/LidoWrapping.sol";
 import "./relayer/VaultActions.sol";
 import "./relayer/VaultPermit.sol";
+import "./relayer/UsdPlusWrapping.sol";
 
 /**
  * @title Batch Relayer Library
  * @notice This contract is not a relayer by itself and calls into it directly will fail.
  * The associated relayer can be found by calling `getEntrypoint` on this contract.
  */
-contract BatchRelayerLibrary is BaseRelayerLibrary, AaveWrapping, LidoWrapping, VaultActions, VaultPermit {
+contract BatchRelayerLibrary is BaseRelayerLibrary, AaveWrapping, LidoWrapping, VaultActions, VaultPermit, UsdPlusWrapping {
```

Comment:
We will likely want a `ERC4246Wrapping` lib that wraps/unwraps ERC4246 tokens

UsdPlus implements the methods necessary to be treated as a ERC4246 by the linear pool, but not the full spec

Reply:
Could we add ERC-4626 `deposit` and `redeem` functions to the UsdPlus wrapper? We could simply forward those calls to the existing `wrap` and `unwrap`, and it would go a long way toward generalizing this PR. We would also have to add the `Deposit` and `Withdraw` events.

Reply:
If @mosinnik is cool with https://github.com/ovnstable/ovnstable-app/pull/13/files than I'm all for it

Reply:
it's ok, now I'm working on implementation deposit and redeem methods near current wrap/unwrap

Reply:
@gtaschuk new methods implemented in https://github.com/ovnstable/ovnstable-app/pull/10, so we ready to rework wrapper to ERC4626